src/prerender.js

```diff
@@ -18,20 +18,17 @@ const FRONT_MATTER_REG = /^\s*---\n\s*([\s\S]*?)\s*\n---\n/i;
 // https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup
 const MAX_DESCRIPTION_LENGTH = 200;
 
-if (process.env.PRERENDER_HOME) {
```

Comment:
AFAIK this is unused? The blame points to a ["wip" commit 3 years ago](https://github.com/preactjs/preact-www/commit/a8957534f632c8ad32ea069c9063828a027bd354) so I went out on a limb and guessed it's not needed.